README.md

```diff
@@ -145,6 +145,8 @@ All variables which can be overridden are stored in [defaults/main.yml](defaults
 | `atom_replication_synchronize_downloads_dir` | False | Boolean variable that enables the synchronization of the downloads directories |
 | `atom_replication_synchronize_uploads_dir` | False | Boolean variable that enables the synchronization of the uploads directories |
 | `atom_replication_ro_nginx_user` | `{% if ansible_os_family == 'RedHat' %}nginx{% elif ansible_os_family == 'Debian' %}www-data{% endif %}` | Nginx user on the AtoM read-only  server. It is needed to run the php symfony commands |
+| `atom_replication_restore_sitetitle` | False | Setting this variable as `True` the role will change the siteTitle on read-only, using the `atom_replication_ro_sitetitle` value |
```

Comment:
When variable is set to `True` the role will change the siteTitle on the read-only bd, using the `atom_replication_ro_sitetitle` value